tests/admin_views/tests.py

```diff
@@ -344,6 +344,21 @@ def test_change_list_sorting_callable(self):
             "Results of sorting on callable are out of order."
         )
 
+    def test_change_list_sorting_property(self):
+        """
+        Ensure we can sort on a list_display field that is property
```

Comment:
Chop _"Ensure we can"_.

Reply:
To add more explanation, docstrings should state the expected behavior and omit prefixes like "Ensure that" since all tests ensure things. This guideline is from [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).

Reply:
Thanks for the pointer. Will do like this in the future (I was just looking example from previous docstrings)